### DISCUSSION TITLE
taskbar wont change

### DISCUSSION THREAD

**Discussion by hellowavingmoto** [valinet/ExplorerPatcher#2029] [General]
![Screenshot 2023-09-09 085430](https://github.com/valinet/ExplorerPatcher/assets/115434223/09e9e4d9-5416-4153-b14a-970f86edd93b)
im trying to set taskbar to windows 10 style but for some reason the task bar wont change

**Comment by Poopooracoocoo** [comment]
You're on an Insider build. You can't expect EP to work properly. This could be https://winclassic.net/thread/1801/legacy-taskbar-removed ([original tweet](https://twitter.com/XenoPanther/status/1687532672644481038)) in effect

**Reply by hellowavingmoto** [reply]
oh ok

**Reply by hellowavingmoto** [reply]
OMG WHY THE HELL THEY REMOVING WIN10 TB?!?!?!

**Reply by pyrates999** [reply]
Please close this.
